crates/autopilot/src/solvable_orders.rs

```diff
@@ -252,6 +252,12 @@ impl SolvableOrdersCache {
 
         let surplus_capturing_jit_order_owners = cow_amms
             .iter()
+            .filter(|cow_amm| {
```

Comment:
Shouldn't we filter the orders instead of the CoW AMM owner addresses? :thinking:

Reply:
I believe cow amm order templates are built in the driver not in the autopilot. The only thing we can do in autopilot is to not send owners for which we know prices are missing.

Reply:
But I don't see how not sending the owners may have an impact. Because we are always sending all the surplus capturing jit order owners independently if the auction contains orders or not. Therefore, if you really want to send only the owners who appear in the auction, you should extend this filter not only for filtering missing prices, but also onwers who don't have orders in the auction :thinking: